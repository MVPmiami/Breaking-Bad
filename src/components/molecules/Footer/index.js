import React from "react";
import styles from "./style.module.scss";

const Footer = () => {
  return (
    <footer>
      <span className={styles.year}>{new Date().getFullYear()}</span>
      <a href="https://github.com/MVPmiami" className={styles.developer}>
        @MVPmiami
      </a>
    </footer>
  );
};

export default Footer;
