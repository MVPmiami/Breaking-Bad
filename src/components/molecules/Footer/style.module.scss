@import "./../../../assets/styles/style.scss";

.footer {
  display: flex;
  justify-content: space-between;
  align-items: center;
  width: calc(100vw - 15rem);
  padding: 0 7.5rem;
  height: 5rem;
  background-color: $footer;
  margin-top: auto;
  overflow-x: hidden;
  font-family: Roboto;
  font-style: normal;
  font-weight: normal;
  .year {
    color: $dataText;
  }
  .developer {
    text-decoration: none;
    color: $dataText;
  }
}
@media (max-width: 610px) {
  .footer {
    width: calc(100vw - 10rem);
    padding: 0 5rem;
  }
}

@media (max-width: 400px) {
  .footer {
    width: calc(100vw - 4rem);
    padding: 0 2rem;
  }
}
