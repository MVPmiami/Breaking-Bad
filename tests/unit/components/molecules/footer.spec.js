import React from "react";
import Footer from "../../../../src/components/molecules/Footer";

const setUp = (props) => shallow(<Footer {...props} />);

describe("should render Footer component", () => {
  let component;
  beforeEach(() => {
    component = setUp();
  });

  it("should contain .cardWrapper wrapper", () => {
    const wrapper = component.find("footer");
    expect(wrapper.length).toBe(1);
    expect(component).toMatchSnapshot();
  });
});
