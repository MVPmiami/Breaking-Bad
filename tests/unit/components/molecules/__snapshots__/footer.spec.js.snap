// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render Footer component should contain .cardWrapper wrapper 1`] = `
<footer>
  <span
    className="year"
  >
    2021
  </span>
  <a
    className="developer"
    href="https://github.com/MVPmiami"
  >
    @MVPmiami
  </a>
</footer>
`;
